docs/quickstart.md

```diff
@@ -1,68 +1,334 @@
+#### Enter the token saved before
+#### You will be asked to save token for later use, enter 'Y'
+
+### If you have removed and re-installed Grafana, you will get an error telling that current API Key is not valid any more
+### You need to enter the new API Key
+
+using API token from config
+.error connect: (401 - 401 Unauthorized) Unauthorized
+enter API token:
+```
 
-Note: the current dashboards specify Prometheus as the datasource. 
-If you use the InfluxDB exporter, you will need to create your own dashboards.
+It will take a few seconds, but at the end, all dashboards will be imported.
 
-## 5. Verify the metrics
+### 9. Verify Dashboards in Grafana
 
-If you use a Prometheus Exporter, open a browser and navigate to [http://0.0.0.0:12990/](http://0.0.0.0:12990/)
-(replace `12990` with the port number of your poller). 
-This is the Harvest created HTTP end-point for your Prometheus exporter. 
-This page provides a real-time generated list of running collectors and names of exported metrics.
+After adding the Prometheus datasource, you can verify that your dashboards are correctly displaying data. Follow these steps:
 
-The metric data that is exported for Prometheus to scrap is 
-available at [http://0.0.0.0:12990/metrics/](http://0.0.0.0:12990/metrics/). 
+1. Open your web browser and navigate to `http://localhost:3000` or `http://<machine-ip>:3000`
+2. Enter your Grafana credentials to log in. The default username is `admin` and the default password is also `admin` (you may have changed this during setup).
+3. Click on the "three lines" button (also known as the hamburger menu) in the top left corner of the Grafana interface. From the menu, select **Dashboards**.
+4. In the Dashboards section, find and click on the **Volume Dashboard**. This dashboard should have been pre-configured or imported as part of your setup. Once the Volume Dashboard is open, you should see various panels displaying data
 
-More information on configuring the exporter can be found in the
-[Prometheus exporter](prometheus-exporter.md) documentation.
+### Troubleshooting
 
-If you can't access the URL, check the logs of your pollers. These are located in `/var/log/harvest/`.
+If you encounter issues, check the logs in `/var/log/harvest` and refer to the [troubleshooting](https://github.com/NetApp/harvest/wiki/Troubleshooting-Harvest) section on the wiki.
+You can also reach out for help on [Discord](https://github.com/NetApp/harvest/blob/main/SUPPORT.md#getting-help) or via email at ng-harvest-files@netapp.com.
 
-## 6. (Optional) Setup Systemd service files
+### Conclusion
 
-If you're running Harvest on a system with Systemd, you may want 
-to [take advantage of systemd instantiated units](https://github.com/NetApp/harvest/tree/main/service/contrib) 
-to manage your pollers.  
+Congratulations! You have successfully set up NetApp Harvest along with Prometheus and Grafana. Enjoy monitoring your systems and feel free to provide feedback.
```

Comment:
🎊 Congratulations! You have successfully set up NetApp Harvest, Prometheus, and Grafana. Enjoy monitoring your systems and feel free to reach out on [Discord, GitHub, or email](https://github.com/NetApp/harvest/blob/main/SUPPORT.md#getting-help).